mapscaleview/src/main/java/com/github/pengrad/mapscaleview/MapScaleView.java

```diff
@@ -121,6 +123,10 @@ protected void onMeasure(int widthMeasureSpec, int heightMeasureSpec) {
         updateScales();
 
         setMeasuredDimension(width, height);
+
+        if (drawer.isExpandLeftEnabled()) {
+            drawer.setExpandLeftStartX(mapScaleModel.getMaxWidth());
```

Comment:
We actually don't need to get `maxWidth` from model, it is just a `width` that was passed there 3 lines before. So we can call `drawer.setViewWidth(width)` or just `setWidth` right after `model.setMaxWidth(width)` (before we call `updateScales`. 
I want we not name it `expandLeftStartX` because for view it is just a width, let's pass it there every time (remove `isExpandLeftEnabled` too) and let drawer decide use it or not. Even for drawer's field name `viewWidth` i think is more suitable than `expandLeftStartX` - in this case we can reuse it for other purposes as is. How you think?

Reply:
It is very sensible to think it that way.
Let's do so.